test/aqua/test_qsvm.py

```diff
@@ -165,8 +166,8 @@ def test_qsvm_binary_directly_statevector(self, use_circuits):
                 except Exception:  # pylint: disable=broad-except
                     pass
 
-    @data(False, True)
-    def test_qsvm_setup_data(self, use_circuits):
+    @data('feature_map', 'circuit', 'library')
```

Comment:
For the tests remember to add logic to suppress the deprecation warning when we know the test is testing deprecated function of Aqua.